package.json

```diff
@@ -86,7 +86,8 @@
   "scripts": {
     "start": "node scripts/start.js",
     "build": "node scripts/build.js",
-    "test": "node scripts/test.js --env=jsdom"
+    "test": "node scripts/test.js --env=jsdom",
+    "deploy": "node scripts/deploy.js"
```

Comment:
What about adding
```
"predeploy": "npm run build",
```
so that you always deploy the latest? or do we consider that risky, if you have uncommitted files in your tree